icanhasdosa.py

```diff
@@ -0,0 +1,19 @@
+from selenium import webdriver as wd 
+from selenium.common.exceptions import NoSuchElementException
+import os
+
+PATH = os.getcwd() + '/chromedriver'
+
+wd = wd.Chrome(PATH)
+wd.implicitly_wait(10)
+
+wd.get("https://suryadarshinicom.wixsite.com/sdhome")
+
+while(True):
+    try: 
+        accept_link = wd.find_element_by_link_text("I ACCEPT")
+        accept_link.click()
```

Comment:
Would this return a response that we need to store for further processing?

Reply:
Not that I'm aware of right now :/

Reply:
I don't believe this function returns anything meaningful